package/com.unity.formats.usd/Runtime/Plugins/x86_64/share/usdAbc/resources/plugInfo.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
Was the alembic plugin moved/removed?

Reply:
Hmmm, I'll have to check my usd build. But it should not. 🤔